tests/org.jboss.tools.openshift.test/src/org/jboss/tools/openshift/test/ui/server/ServerSettingsWizardPageModelTest.java

```diff
@@ -94,6 +95,17 @@ public void shouldReturnProjectThatHasGitRemoteMatchingBuildConfig() {
 		assertThat(project).isEqualTo(project2);
 	}
 
+	@Test
+	public void testPodPathProperty() {
```

Comment:
Yes, because useInferredPodPath and podPath parameters were not assigned to class fields in update().